string hand # left or right
string state # hand-object contact info
float32 score # score
jsk_recognition_msgs/Rect hand_rect # hand rect
jsk_recognition_msgs/Rect object_rect # object rect in contact
